.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+    - "3.6"
+
+services:
+    - docker
+
+install:
+    - pip install flake8
+    - docker build . -t baselines-test
+
+script:
+    - flake8 --select=F common
```

Comment:
How does this run against baselines/common instead of common?

Reply:
good point, it should be baselines/common. Fixed now, it still should run clean.